crates/secrets-core/src/location.rs

```diff
@@ -0,0 +1,553 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache License, Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024 Datadog, Inc.
+
+use crate::common::ByteSpan;
+use bstr::{BStr, ByteSlice};
+use std::cell::{Cell, RefCell};
+use std::num::NonZeroU32;
+
+/// A one-based point representing a line and column in a string.
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+pub(crate) struct Point {
+    pub line: NonZeroU32,
+    pub col: NonZeroU32,
+}
```

Comment:
Why not reusing the `Position` from the [common model](https://github.com/DataDog/datadog-static-analyzer/blob/main/crates/static-analysis-kernel/src/model/common.rs#L96)?